packages/manager/apps/hpc-vmware-managed-vcd/src/pages/listing/organizations/Organizations.page.tsx

```diff
@@ -81,7 +81,7 @@ export default function Listing() {
 
   const datagridProps: TDatagridContainerProps = {
     title: t('managed_vcd_listing_title'),
-    containerId: organizationListingContainerId,
+    queryKey: [organizationListingContainerId],
```

Comment:
`organizationListingContainerId` should be renamed to `organizationListingQueryKey`